### PR TITLE
[DOC] Correct directory structure

### PR DESCRIPTION
**Pull request by wvandertoorn**
Include all files and their extensions.

### CODE REVIEW

**Comment by smehringer** [comment]
@wvandertoorn Next time you do not need to close the PR and reopen another one :) You can just `push` your changes to the same branch, and github will automatically show the updates in the same PR.

**Comment by smehringer** [comment]
@joergi-w You can add a card in the Documentation project with an example and a description if you want to keep track of this without tackling it right away :)

**Comment by joergi-w** [comment]
Done, thanks!

**Comment by wvandertoorn** [comment]
> @wvandertoorn Next time you do not need to close the PR and reopen another one :) You can just `push` your changes to the same branch, and github will automatically show the updates in the same PR.

Check, will do next time!

File: doc/setup/quickstart_cmake/index.md [hunk 1]
```diff
@@ -81,14 +81,17 @@ tutorial
...
     ├── include
     ├── submodules
-    └── test
+    ├── test
+    ├── CHANGELOG.md
+    ├── CMakeLists.txt
+    ├── CODE_OF_CONDUCT.md
+    ├── CONTRIBUTING.md
+    ├── LICENSE.md
+    └── README.md
```
Comment (eseiler) [inline doc/setup/quickstart_cmake/index.md]: The ordering doesn't look like what `tree` or `ls` would report?

```cpp
$ tree -L 1 seqan3
seqan3
├── CHANGELOG.md
├── CMakeLists.txt
├── CODE_OF_CONDUCT.md
├── CONTRIBUTING.md
├── LICENSE.md
├── README.md
├── build_system
├── doc
├── include
├── submodules
└── test
```

```bash
$ ls -l
total 44
<metainfo> CHANGELOG.md
<metainfo> CMakeLists.txt
<metainfo> CODE_OF_CONDUCT.md
<metainfo> CONTRIBUTING.md
<metainfo> LICENSE.md
<metainfo> README.md
<metainfo> build_system
<metainfo> doc
<metainfo> include
<metainfo> submodules
<metainfo> test
```

Comment (smehringer) [reply doc/setup/quickstart_cmake/index.md]: I would add the command `tree -L 1 seqan3` just before showing the directory structure itself. 
So the user can call the same command and directly see if everything is up to date :)
(And then adjusting the output like @eseiler suggested)

Comment (wvandertoorn) [reply doc/setup/quickstart_cmake/index.md]: I included the ``` tree ``` command. At this point of the tutorial, the user is in the ``` tutorial ``` directory, and I think that double checking if the ``` source ``` and ``` build ``` directories were successfully created too, is a good idea. Therefore, I used ``` tree -L 2 ```. Also, when running this command, I get another tree structure than suggested above, I've used the one I got.

Comment (smehringer) [reply doc/setup/quickstart_cmake/index.md]: Either I am blind or I do not see a `build` directory in your current update :eyes:  Cna you double check?

Comment (wvandertoorn) [reply doc/setup/quickstart_cmake/index.md]: It's in there :) line 81

Comment (smehringer) [reply doc/setup/quickstart_cmake/index.md]: got it :woman_facepalming:

**Pull request merged**